tests/saitests/py3/sai_qos_tests.py

```diff
@@ -1246,16 +1247,18 @@ def runTest(self):
             xmit_counters, _ = sai_thrift_read_port_counters(
                 self.dst_client, asic_type, port_list['dst'][dst_port_id])
             test_stage = 'after send a few packets to trigger drop'
-            sys.stderr.write('{}:\n\trecv_counters {}\n\trecv_counters_base {}\n\t' +
+            sys.stderr.write('{}:\n\trecv_counters {}\n\trecv_counters_base {}\n\t'
                              'xmit_counters {}\n\txmit_counters_base {}\n'.format(                           # noqa F841
                                  test_stage, recv_counters, recv_counters_base, xmit_counters, xmit_counters_base))
             # recv port pfc
             assert(recv_counters[pg] > recv_counters_base[pg]), \
                 'unexpectedly PFC counter not increase, {}'.format(test_stage)
             # recv port ingress drop
-            if platform_asic and platform_asic == "broadcom-dnx":
+            if (platform_asic and platform_asic == "broadcom-dnx") or \
+                    self.hwsku in ['Cisco-8800-LC-48H-C48']:
                 logging.info(
-                    "On J2C+ don't support port level drop counters - so ignoring this step for now")
+                    "On J2C+ and cisco-8000-Q100 don't support port level drop"
+                    " counters - so ignoring this step for now")
```

Comment:
can we move this check below to skip only recv counter validation.  tx counter check should still work.

rx_counter  check is skipped since we don;t have per port counter for this.

Reply:
Verified that tx is working, so re-enabling the check. rx is disabled for Pacific platforms.